modules/owncloud_web/pages/index.adoc

```diff
@@ -1,10 +1,41 @@
-= Introduction
+= ownCloud Web
 
-ownCloud Web
+:toc: right
+:toc-levels: 1
+
+:description:
+ownCloud web comes with a completely new design that is more user-friendly and intuitive and, of course, optimized for ownCloud Infinite Space (oCIS).
+
+== Introduction
+
+{description}
+
+If you are still running ownCloud 10.x as server, consider installing the {oc-marketplace-url}/apps/web[Web] app, which provides the new look and feel plus some additional features, but not everything ownCloud Web has to offer when used with oCIS. For a quick introduction about what's new or different, check out the section xref:web_with_oC10.adoc[ownCloud Web on ownCloud Server 10.x].
 
 // Test for oCIS
 
 // [source,yaml]
 // ----
 // include::https://raw.githubusercontent.com/wkloucek/cst-workshop/main/workshop-2022-02-02-docker/solution/docker-compose.yml[]
 // ----
+
+// I guess the above can be deleted now?
+
+== Changes
+
+For readers already familiar with ownCloud 10 and its standard web interface, the big differences in the new ownCloud Web on oCIS are:
+
+* A different sharing concept has been implemented,
+
```

Comment:
linebreaks in list items are not necessary, but you can keep them if you desperately want them 😅

Reply:
> linebreaks in list items are not necessary, but you can keep them if you desperately want them sweat_smile

Let's weed them out with the next revision.